bundletester/tester.py

```diff
@@ -83,6 +83,9 @@ def configure():
                         help='A plan to deploy charm under')
     parser.add_argument('--deploy-budget',
                         help='Deploy budget and allocation limit')
+    parser.add_argument('--no-chaos', action="store_true",
```

Comment:
I had this idea that "no chaos" was friendlier, because you didn't have to know that matrix was called matrix.

But ... yeah, that makes no sense. Flag renamed :-)